include/zephyr/drivers/i3c.h

```diff
@@ -35,27 +35,34 @@ extern "C" {
  *   - 1: I3C Controller capable
  *   - 2: Reserved
  *   - 3: Reserved
+ *   .
```

Comment:
This is weird that this would be needed, which version of doxygen are you using? I can't seem to reproduce the warning? (1.9.6 here)

Reply:
CI is using 1.9.6. I don't think we necessarily "depend" on a newer version if the only difference is warnings in one and none in the other, but agreed this need to be clearer. Also there is also the opposite issue which is that IIRC generation fails for the most recent 1.9.8 (can't remember if this is "just" Breathe or in Doxygen proper)

Reply:
Some Doxygen versions (e.g. 1.9.5) had bugs that made them unusable for Zephyr. Maybe 1.9.1 has some bugs as well.